xml/System.Net.Sockets/NetworkStream.xml

```diff
@@ -410,7 +410,7 @@
         <param name="buffer">An array of type <see cref="T:System.Byte" /> that is the location in memory to store data read from the <see cref="T:System.Net.Sockets.NetworkStream" />.</param>
         <param name="offset">The location in <paramref name="buffer" /> to begin storing the data.</param>
         <param name="size">The number of bytes to read from the <see cref="T:System.Net.Sockets.NetworkStream" />.</param>
-        <param name="count">To be added.</param>
+        <param name="count">The number of bytes to read from the <see cref="T:System.Net.Sockets.NetworkStream" />.</param>
```

Comment:
This reads exactly the same as the `size` parameter. Same in the other 5 places below.

Reply:
Nevermind, I see this is because the parameter name changed. I didn't see a breaking change come through for these name changes.